packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -148,7 +152,9 @@ const PostsPagePostHeader = ({post, classes}: {
           {!post.isEvent && <span className={classes.date}>
             <PostsPageDate post={post} hasMajorRevision={hasMajorRevision} />
           </span>}
-          {post.types && post.types.length > 0 && <Components.GroupLinks document={post} />}
+          {post.isEvent && <div className={classes.groupLinks}>
+            <Components.GroupLinks document={post} />
+          </div>}
```

Comment:
@jpaddison3 I guess LW agreed with you :)

<img width="700" alt="Screen Shot 2021-11-09 at 10 09 46 AM" src="https://user-images.githubusercontent.com/9057804/140952315-9e2ff293-0a14-4a79-8876-5ca87ecba438.png">
<img width="400" alt="Screen Shot 2021-11-09 at 10 10 03 AM" src="https://user-images.githubusercontent.com/9057804/140952352-b7df378c-7d07-45b2-95a8-44f5be3caa90.png">

This is the typical case, just having one external link:
<img width="700" alt="Screen Shot 2021-11-09 at 10 10 35 AM" src="https://user-images.githubusercontent.com/9057804/140952412-07fddd9d-bdf6-46e1-9447-916ff2852d02.png">
<img width="400" alt="Screen Shot 2021-11-09 at 10 10 21 AM" src="https://user-images.githubusercontent.com/9057804/140952438-9e60f8e1-c3fa-40d8-9295-63c288c72437.png">

I actually don't really like any of the placements so far, so I'm fine changing it. I toyed around with replacing the karma on the right side with the icons to visually separate them from the post content (kind of like how [this page](https://betterlesson.com/strategy/120/planning-a-pbl-design-thinking-backwards-planning-for-teachers) handles social icons) but I didn't find a design I was happy with, and I figured it's not worth taking much time now so we can revisit it later.

Reply:
I think there's a problem of excessive & inconsistent whitespace between the author and the link icons. You should probably have the marginLeft on the groupLinks div be dynamic based on whether there's document.types. And the first child of groupLinks should not have margin on it.

I would also make sure to test with an event associated with a type (LW, SSC, etc). You'll need to turn on LW-mode to do so.